### PR TITLE
DEVEXP-248: Fix webhooks models

### PR DESCRIPTION
**Pull request by asein-sinch**

### CODE REVIEW

File: packages/voice/src/models/v1/ace-request/ace-request.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { AceRequestAmd } from '../ace-request-amd';
 export interface AceRequest {
 
   /** Must have the value `ace`. */
-  event?: string;
+  event?: 'ace';
```
Comment (Dovchik) [inline packages/voice/src/models/v1/ace-request/ace-request.ts:16]: As it has value, remove optionality?
```suggestion
  event: 'ace';
```

File: packages/voice/src/models/v1/dice-request/dice-request.ts [hunk 2]
```diff
@@ -9,22 +9,23 @@ import { CallHeader } from '../call-header';
...
   /** Must have the value `dice`. */
-  event?: string;
+  event?: 'dice';
   /** The unique ID assigned to this call. */
   callId?: string;
   /** The timestamp in UTC format. */
   timestamp?: Date;
   /** The reason the call was disconnected. */
-  reason?: string;
+  reason?: GetCallResponseObjReasonEnum;
```
Comment (Dovchik) [inline packages/voice/src/models/v1/dice-request/dice-request.ts:26]: What about more generalized name for enum? e.g.
```suggestion
  reason?: CallResultReasonEnum;
```

File: packages/voice/src/models/v1/dice-request/dice-request.ts [hunk 3]
```diff
@@ -9,22 +9,23 @@ import { CallHeader } from '../call-header';
...
   /** The result of the call. */
-  result?: string;
+  result?: GetCallResponseObjResultEnum;
```
Comment (Dovchik) [inline packages/voice/src/models/v1/dice-request/dice-request.ts:28]: Same
```suggestion
  result?: CallResultEnum;
```

Comment (asein-sinch) [reply packages/voice/src/models/v1/dice-request/dice-request.ts:28]: Same pattern because of same source in specification

**Review by JPPortier — APPROVED** [review]

**Pull request merged**